### PR TITLE
Fix puppet strings warnings and minor README.md update

### PR DESCRIPTION
**Pull request by vinzent**
Puppet 3.8.7 was specified in README.md but this is now Puppet 4 only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,14 @@ selinux::boolean { 'puppetagent_manage_all_files': }
     * `semanage` requires `--noreload` while in disabled mode when
       adding or changing something
     * Only few `--list` operations work
+* run acceptance tests:
+
+```
+BEAKER_debug=yes BEAKER_set="centos-6-x64" PUPPET_INSTALL_TYPE="agent" be rake beaker
```
Comment (bastelfreak) [inline README.md]: not everybody uses this alias. should we use bundle exec instead?

**Comment by vinzent** [comment]
added the centos 6 and 7 nodesets too. PR open in modulesync_config: https://github.com/voxpupuli/modulesync_config/pull/313

**Comment by vinzent** [comment]
don't merge.

**Comment by vinzent** [comment]
sorry, I thought the docs/ folder was messed up. but it was only my try to run `puppet strings server` which doesn't do what i've expected.

**Pull request merged**